test/bun.js/console/console-log.test.ts

```diff
@@ -0,0 +1,18 @@
+import { file, spawn } from "bun";
+import { expect, it } from "bun:test";
+import { bunExe } from "bunExe";
+
+it("should log to console correctly", async () => {
+  const { stdout, stderr, exited } = spawn({
```

Comment:
if you just want to test console.log

We can use `Bun.inspect` for this

`Bun.inspect` runs the same code as console.log but outputs to a string